kedro/framework/cli/starters.py

```diff
@@ -149,11 +295,21 @@ def starter():
 @starter.command("list")
 def list_starters():
     """List all official project starters available."""
-    repo_url = _STARTERS_REPO.replace("git+", "").replace(".git", "/tree/main/{alias}")
-    output = [
-        {alias: repo_url.format(alias=alias)} for alias in sorted(_STARTER_ALIASES)
-    ]
-    click.echo(yaml.safe_dump(output))
+    starters_aliases = _get_starters_aliases()
+
+    # ensure kedro starters are listed first
+    official_starters = sorted(
+        list(filter(lambda x: x.origin == "kedro", starters_aliases))
```

Comment:
```suggestion
        starter_alias for starter_alias in starters_aliases if starter_alias.origin != "kedro"
```
Generally we avoid too much functional stuff in the kedro codebase in favour of comprehensions 🙂

Reply:
I found `filter` is quite expressive in this case with a straightforward lambda, but no strong preference here, I am okay with switching it to list comprehension.